### PR TITLE
Support for non-Ethereum stablecoins

### PR DESCRIPTION
**Pull request by jazg**
Add support for non-Ethereum stablecoins such as USDC/USDT on Polygon/Avalanche. This PR also defines some new types for token assets on Goerli.

### CODE REVIEW

File: multichain.go [hunk 1]
```diff
@@ -211,30 +213,28 @@ func (asset Asset) OriginChain() Chain {
...
 
 	case BADGER, BUSD, CRV, DAI, EURT, FTT, ibBTC, KNC, LINK, MIM, REN, ROOK,
 		SUSHI, UNI, USDC, USDT:
 		return Ethereum
 
-	case KETH:
-		return Kovan
-	case GETH:
+	case DAI_Goerli, REN_Goerli, USDC_Goerli, USDT_Goerli:
 		return Goerli
```
Comment (tok-kkk) [inline multichain.go:238]: Maybe this can be combined with `GETH` like what we do to `Polygon` and `Avalanche`

File: multichain.go [hunk 2]
```diff
@@ -284,16 +282,14 @@ func (asset Asset) ChainType() ChainType {
 // Type returns the asset-type (Native or Token) for the given asset.
 func (asset Asset) Type() AssetType {
```
Comment (tok-kkk) [inline multichain.go:283]: This function only works for Evm assets and erc20s, wonder if this is intentional?

**Review by tok-kkk — CHANGES_REQUESTED** [review]

File: multichain.go (hunk 2 above)
Comment (jazg) [reply multichain.go:294]: I think it's more that it's only necessary to differentiate for chains that support multiple kinds of assets.

Comment (tok-kkk) [reply multichain.go:294]: Got it.

**Review by tok-kkk — APPROVED** [review]
approved

**Pull request merged**